packages/test/src/test-type-helpers.ts

```diff
@@ -0,0 +1,143 @@
+import vm from 'vm';
```

Comment:
Can you test that if you define the same class twice in the same context with the same symbol, instanceof works?